### PR TITLE
Handle hedera_signAndReturnTransaction request

### PR DESCRIPTION
**Pull request by huck-hgraph**
### Summary
- Adds `signAndReturnTransaction` method to `HederaWallet`
- Displays transaction information in session request modal

https://github.com/hgraph-io/hedera-walletconnect-wallet/assets/136644362/1abb1b21-8563-4ef9-845d-a8e03c603181

### CODE REVIEW

File: src/lib/HederaLib.ts [hunk 1]
```diff
@@ -51,4 +51,20 @@ export class HederaWallet {
       return { error: (e as Error).message }
     }
   }
+
+  public async signAndReturnTransaction(transaction: Transaction) {
+    try {
+      const signedTransaction = await transaction.sign(this.privateKey)
+      const signedTransactionBytes = signedTransaction.toBytes()
+      const encodedTransactionBytes = Buffer.from(signedTransactionBytes).toString('base64')
+      return {
```
Comment (huck-hgraph) [inline src/lib/HederaLib.ts:60]: @HGraphPunks @hgraphql LMK what you think of the format here. Could also just be:
```js
{
  transactionBytes: encodedTransactionBytes
}
```

Comment (HGraphPunks) [reply src/lib/HederaLib.ts:60]: I think this is cleaner for now. If we add type, which isn't defined currently, would you think transaction.bytes and tranaction.type is cleaner for expanding the json in the future

or is just adding transactionType in the root a solution that'd make it easier? I feel like there's no right or wrong, just a pattern to set.

Comment (huck-hgraph) [reply src/lib/HederaLib.ts:60]: Good point, I pushed a change to include transaction type in the returned result. Now the params for the request and result are the same:
```js
{
  transaction: {
    type: string
    bytes: string
  }
}
```

Comment (hgraphql) [reply src/lib/HederaLib.ts:60]: nice, yeah I think this looks clean

**Review by hgraphql — APPROVED** [review]
nice

**Pull request merged**